server/server_test.ts

```
@@ -15,6 +15,7 @@ invokeTest(
     const response = await fetch(`${host}/invoke/up`, {
       method: "POST",
       body: JSON.stringify("str"),
+      headers: { "Yext-Invocation-ID": "custom-invocation-id" },
```

Is there any difference in behavior that could be asserted for this test?

---

Eh, this test would need to be expanded substantially to exercise fetching; I just added this header to ensure it was being hit in the form the actual client will use.

I did update the fetch test case though, which should at least catch everything except passing the header from the server to `PluginHost`.